subsys/bluetooth/mesh/beacon.c

```diff
@@ -96,12 +96,15 @@ static bool secure_beacon_send(struct bt_mesh_subnet *sub, void *cb_data)
 	uint32_t now = k_uptime_get_32();
 	struct net_buf *buf;
 	uint32_t time_diff;
+	uint32_t time_diff_observed;
```

Comment:
```suggestion
	uint32_t time_since_last_recv;
```
or something, just to make this name a little less generic?